pkg/webhook/webhook.go

```diff
@@ -621,6 +622,60 @@ func getResourceList(resourceRequests map[string]int64) *corev1.ResourceList {
 	return &resourceList
 }
 
+func createCustomizedPatch(pod corev1.Pod) ([]jsonPatchOperation, error) {
+	var patch jsonPatchOperation
+	var customizedPatch []jsonPatchOperation
+
+	for cik, civ := range customizedInjections {
+		// The customizedInjections will be injected when:
+		// 1. Pod labels contain the patch key defined in customizedInjections, and
+		// 2. The value of patch key in pod labels(not in customizedInjections) is "true"
+		if podValue, exists := pod.ObjectMeta.Labels[cik]; exists && podValue == "true" {
+			err := json.Unmarshal([]byte(civ), &patch)
+			if err != nil {
+				return nil, err
+			}
+		}
+		customizedPatch = append(customizedPatch, patch)
```

Comment:
should "customizedPatch = append(customizedPatch, patch)" be within the previous if statement closure - before ln 638, so we only add new patches?

Reply:
Yes, it should.
Done